packages/office-ui-fabric-react/src/components/SelectedItemsList/SelectedPeopleList/Items/EditingItem.tsx

```diff
@@ -1,38 +1,79 @@
 /* tslint:disable */
 import * as React from 'react';
 /* tslint:enable */
-import { BaseComponent, KeyCodes, getId, getNativeProps, inputProperties, css } from '../../../../Utilities';
+import { KeyCodes, getId, getNativeProps, inputProperties, css } from '../../../../Utilities';
 import { FloatingPeoplePicker, IBaseFloatingPickerProps } from '../../../../FloatingPicker';
-import { ISelectedPeopleItemProps } from '../SelectedPeopleList';
 import { IExtendedPersonaProps } from '../SelectedPeopleList';
 import { IPeoplePickerItemState } from './ExtendedSelectedItem';
 import { IPersonaProps } from '../../../../Persona';
 
-import * as stylesImport from './EditingItem.scss';
-
-// tslint:disable-next-line:no-any
-const styles: any = stylesImport;
-
-export interface IEditingSelectedPeopleItemProps extends ISelectedPeopleItemProps {
-  // tslint:disable-next-line:no-any
-  onEditingComplete: (oldItem: any, newItem: any) => void;
-  onRenderFloatingPicker?: React.ComponentType<IBaseFloatingPickerProps<IPersonaProps>>;
-  floatingPickerProps?: IBaseFloatingPickerProps<IPersonaProps>;
-  getEditingItemText?: (item: IExtendedPersonaProps) => string;
+import * as styles from './EditingItem.scss';
+
+export interface IEditingItemProps<TItem> extends React.HTMLAttributes<any> {
+  /**
+   * The current item of the EditingItem
+   */
+  item: TItem;
+
+  /**
+   * Callback for when the edited item's new value has been selected.
+   * Invoked indirectly by the picker mounted by onRenderFloatingPicker.
+   */
+  onEditingComplete: (oldItem: TItem, newItem: TItem) => void;
```

Comment:
This is types tightening over the previous `any`. This should be ok as long as we're sure that `IPersonaProps` was the only valid type previously.

Reply:
Yeah, `onEditingComplete` is only called in EditingItem._onSuggestionSelected, was itself typed against the extended persona type.

https://github.com/OfficeDev/office-ui-fabric-react/blob/7020ac61c62e19e3a7ca3ca601a047f6d288f560/packages/office-ui-fabric-react/src/components/SelectedItemsList/SelectedPeopleList/Items/EditingItem.tsx#L122